### PR TITLE
Bump build image to 1.56.1

### PR DESCRIPTION
**Pull request by jamwaffles**

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ci_build_image := "jamwaffles/circleci-embedded-graphics:1.40.0-3"
+ci_build_image := "jamwaffles/circleci-embedded-graphics:1.56.1"
```
Comment (rfuest) [inline justfile]: I would always add the suffix to keep the same tag format.
```suggestion
ci_build_image := "jamwaffles/circleci-embedded-graphics:1.56.1-0"
```

**Review by rfuest — CHANGES_REQUESTED** [review]
One small nit

File: justfile (hunk 1 above)
Comment (jamwaffles) [reply justfile]: I started with `-0` and decided to remove it, but yeah let's keep it for consistency 👍

**Review by rfuest — APPROVED** [review]

**Comment by jamwaffles** [comment]
Thanks :)

**Pull request merged**